libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/db/DataSourceFactory.java

```diff
@@ -47,13 +47,15 @@
     static final String MYSQL_VALIDATION_QUERY = "/* ping */ SELECT 1";
 
     static final String DRIVER_CLASS_NAME = "athenz.db.driver.class";
+    static final String AWS_SECRETS_MNGR_DRIVER_CLASS_NAME = "com.amazonaws.secretsmanager.sql.AWSSecretsManagerMySQLDriver";
     public static PoolableDataSource create(String url, Properties mysqlConnectionProperties) {
         
         String driver = null;
         try {
             if (url.indexOf(":mysql:") > 0) {
                 
                 driver = System.getProperty(DRIVER_CLASS_NAME, "com.mysql.cj.jdbc.Driver");
+                url = manipulateConnectionUrl(url, driver);
```

Comment:
@dvirguttman why do we need the changes in this file? wouldn't it be my responsibility if I want to change the driver property to the com.amazonaws.secretsmanager.sql.AWSSecretsManagerMySQLDriver, also make sure the url is named accordingly?